README.md

```diff
@@ -60,6 +60,7 @@ type XhrOptions = String | {
     withCredentials: Boolean?,
     responseType: String?,
     beforeSend: Function?
+    qs: Any?
```

Comment:
I disagree here, the type of `qs` is unknown until you specify the `xhr.queryStringStringify` or `xhr.qs` function. 
You can specify a function accepting an array or an object or a specific structure or a Map...